product-mini/platforms/common/wasm_proposal.c

```diff
@@ -5,6 +5,8 @@
 
 #include <stdio.h>
 
+#include "../../../core/config.h"
```

Comment:
```suggestion
#include "bh_platform.h"
```

Usually, we `#include "bh_platform.h"` which `#include "../platform/include/platform_common.h"` which `#include "../../../config.h"`

Reply:
Done, thanks